man/systemctl.xml

```diff
@@ -2136,6 +2136,17 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--timestamp</option></term>
+
+        <listitem>
+          <para>Takes one of <literal>pretty</literal> (the default),
+          <literal>us</literal>, <literal>µs</literal>. Changes the format of printed timestamps.
+          <literal>pretty</literal>: <literal>Day YYYY-MM-DD HH:MM:SS TZ</literal>
+          <literal>us</literal>: <literal>Day YYYY-MM-DD HH:MM:SS.UUUUUU TZ</literal></para>
```

Comment:
maybe if we add this also add UTC style right away, I am pretty sure this will come next

Reply:
ok, done